api/python/quilt3/telemetry.py

```diff
@@ -0,0 +1,159 @@
+import atexit
+from concurrent.futures import ThreadPoolExecutor, wait
+import functools
+import os
+import platform
+from requests_futures.sessions import FuturesSession
+import sys
+from threading import Lock
+import uuid
+
+from .util import load_config, set_config_value
+from . import __version__ as quilt3_version
+
+TELEMETRY_URL = "https://telemetry.quiltdata.cloud/Prod/metrics"
+TELEMETRY_USER_AGENT = "QuiltCli"
+TELEMETRY_CLIENT_TYPE = "quilt3-python-client"
+TELEMETRY_SCHEMA_VERSION = "pyclient-usage-metrics-v0"
+
+DISABLE_USAGE_METRICS_ENVVAR = "QUILT_DISABLE_USAGE_METRICS"
+MAX_CLEANUP_WAIT_SECS = 5
+
+
+
+
+class ApiTelemetry:
+    session = None
+    session_id = str(uuid.uuid4())
+    pending_reqs = []
+    pending_reqs_lock = Lock()
+    telemetry_disabled = False
```

Comment:
Um... this needs to be initialized to `None` - otherwise `if ApiTelemetry.telemetry_disabled is None:` doesn't work?